tests/playwright/pages/editor-page.ts

```diff
@@ -537,7 +537,7 @@ export default class EditorPage extends BasePage {
 	 */
 	async editCurrentPage() {
 		const postId = await this.getPageIdFromFrontEnd();
-		await expect( postId, 'No Post/Page ID returned when calling getPageIdFromFrontEnd().' ).toBeTruthy();
+		await expect.soft( postId, 'No Post/Page ID returned when calling getPageIdFromFrontEnd().' ).toBeTruthy();
```

Comment:
I have replaced `expect()` with `expect.soft()` everywhere, so that it will give all errors in a test all at once. This is helpful e.g. when making structural updates to tests.

Reply:
Please don't do it. It probably makes sense with screenshot, but not with e2e. Very often one step depends on another and if it was failing in step 1, playwright will fail fast and retry. But if you have soft assertions, it will go over broken test and only then will do a retry. It's a standard practise on most of the test automation frameworks.  Also, it is not a part of this PR. Such changes should come as separate